src/utils/params.ts

```diff
@@ -24,18 +21,18 @@ export function updateUrlCode(code: string) {
 
 export function getParamsFromQuery(location = history.location) {
   try {
-    return queryString.parse(
+    return new URLSearchParams(
       playroomConfig.paramType === 'hash'
         ? location.hash.replace(/^#/, '')
         : location.search
     );
   } catch (err) {
-    return {};
+    return new URLSearchParams();
```

Comment:
This `try catch` might not even be needed anymore since AFAICT URLSearchParams constructor never throws. I couldn't make it throw on any input, and a cursory glance at the spec didn't mention throwing. But probably better to be safe than sorry.